functional-tests/tests/test_anonymizer.py

```diff
@@ -18,7 +18,10 @@ def test_anonymize():
     response_status, response_content = anonymize(request_body)
 
     assert response_status == 200
-    assert response_content == "hello world, my name is ANONYMIZED. My number is: 034453334"
+    assert (
+        response_content
```

Comment:
I must say this lint is really weird and unclear.

Reply:
Will try to see how to exclude.. I think that adding a backslash might help as well